modules/ROOT/pages/dataweave-troubleshoot.adoc

```diff
@@ -264,5 +264,48 @@ read(payload.^raw replace "\uFFFF" with "NonChar")
 
 == Low Performance Issues
 
+=== application/dw format
 Using `output application/dw` format can impact the performance of transformations. This format is intended to help you debug the results of DataWeave transformations.
 It is significantly slower than other formats, so avoid using this format in production applications.
+
+=== multipart/form-data content parsing
+When working with `multipart/form-data` inputs, trying to access the content of a part can cause performance degradation, especially for large parts. If the intention is not to parse the contents, use the `^raw` selector to avoid parsing.
```

Comment:
Can we explain a bit more about why this could happen? Otherwise, it sounds like multipart will always have issues. I'd be clear around this occurring because we parse the parts in formats we understand so they can be queried and analyzed, but if that's not a requirement and they can be treated as binary content then raw will work. I think we can also make a note that this is particularly relevant when dealing with formats with high memory requirements like Excel files.

Reply:
Addressed in the new revision